src/modules/45-dashboards/pages/home/UpdateDashboardForm.tsx

```diff
@@ -26,7 +27,12 @@ interface UpdateDashboardFormProps {
   }
 }
 
-const UpdateDashboardForm: React.FC<UpdateDashboardFormProps> = ({ hideModal, reloadDashboards, formData }) => {
+const UpdateDashboardForm: React.FC<UpdateDashboardFormProps> = ({
+  editableFolders = [],
```

Comment:
Looks like it's not optional here too
```suggestion
  editableFolders,
```